### ISSUE TITLE
write the result of the query on each of the lines

### ISSUE THREAD

**Issue by omidn** [s1na/sc2map#12]
we should get the analytics results for each of the steps of the process and write that info in the bottom of the line that depicts it on the map.

**Comment by acid0078** [comment]
The result is currently displayed within a tooltip. It should be just a number next to the line. No further information need, just the raw value (because it just appears after selecting a respective metric).

**Comment by omidn** [comment]
yeah, that sounds perfect. 

tried this one but couldn't get it working yet
https://github.com/Leaflet/Leaflet.label

**Comment by omidn** [comment]
we do it somehow now..

**Comment by acid0078** [comment]
I don't think this is closed. He explicitly wanted the "results to be shown with a number directly attached to the lines". Our tooltip also adds the results each time we do a new analysis. We should make it just removes the old result and adds a new one.
